BigBaseV2/src/views/vehicle/view_lsc.cpp

```diff
@@ -176,23 +205,17 @@ namespace big
 
 		if (ImGui::Checkbox("Bulletproof Tires", &can_tires_burst))
 		{
-			g_fiber_pool->queue_job([] {
-				VEHICLE::SET_VEHICLE_TYRES_CAN_BURST(player_vehicle, !can_tires_burst);
-			});
+			VEHICLE::SET_VEHICLE_TYRES_CAN_BURST(player_vehicle, !can_tires_burst);
```

Comment:
Calling natives outside of script threads is detected.

Use the fiber pool please.